src/Installers/Rpm/Directory.Build.targets

```diff
@@ -85,7 +85,7 @@
       <FpmArgs Include="&quot;$(PackageContentRoot)=$(RpmPackageInstallRoot)&quot;" />
     </ItemGroup>
 
-    <Exec Command="scl enable rh-ruby25 'fpm @(FpmArgs,' ')'" />
+    <Exec Command="scl enable rh-ruby26 'fpm @(FpmArgs,' ')'" />
```

Comment:
Why this update?

Reply:
Ruby in the image was updated to ruby26 in this PR: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/700 because of this issue: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/699